client/src/components/Home/Home.module.css

```diff
@@ -0,0 +1,4 @@
+.screenContainer {
+    text-align: center;
+    padding: 17%;
```

Comment:
% is very bad in the alignment of elements because it changes as the screen size changes try using px or rm